### PR TITLE
Postal code level data export

### LINKED ISSUE #229: Create postal number level open data
The postal number are data set should contain same data as is in city level data set. And:
- Remove all postal number areas with less than 10 replies. 
- Remove all non-valid postal number areas.
- Remove postal number areas with population smaller than 500
- Do not include ages or any other personal data
- Include the population and the name of the postal number area

### PR DESCRIPTION
**Pull request by cutepig**
Closes #229 

Adds accumulated general results by postal code as `postalcode_level_general_results.json`.

Generated example: https://data.dev.oiretutka.fi/postalcode_level_general_results.json

### CODE REVIEW

File: src/backend/cli.ts [hunk 1]
```diff
@@ -52,33 +71,46 @@ yargs
     },
     async args => {
       try {
-        await dump(args as any);
+        await dumpCommand(args as any);
```
Comment (nathandao) [inline src/backend/cli.ts:74]: :+1:  : D

File: src/backend/dataExports/cityLevelWeeklyGeneralResults.ts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  COUNT(IF ( headache = 'yes', 1, NULL)) AS headache_yes,
+  COUNT(IF ( sore_throat = 'no', 1, NULL)) AS sore_throat_no,
+  COUNT(IF ( sore_throat = 'yes', 1, NULL)) AS sore_throat_yes,
+  COUNT(IF ( rhinitis = 'no', 1, NULL)) AS rhinitis_no,
+  COUNT(IF ( rhinitis = 'yes', 1, NULL)) AS rhinitis_yes,
+  COUNT(IF ( stomach_issues = 'no', 1, NULL)) AS stomach_issues_no,
+  COUNT(IF ( stomach_issues = 'yes', 1, NULL)) AS stomach_issues_yes,
+  COUNT(IF ( sensory_issues = 'no', 1, NULL)) AS sensory_issues_no,
+  COUNT(IF ( sensory_issues = 'yes', 1, NULL)) AS sensory_issues_yes,
+  COUNT(IF ( longterm_medication	 = 'no', 1, NULL)) AS longterm_medication_no,
```
Comment (nathandao) [inline src/backend/dataExports/cityLevelWeeklyGeneralResults.ts:53]: Do the extra spaces come from auto-formatting? : D

File: src/backend/dataExports/postalCodeLevelGeneralResults.ts [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+  COUNT(IF ( headache = 'yes', 1, NULL)) AS headache_yes,
+  COUNT(IF ( sore_throat = 'no', 1, NULL)) AS sore_throat_no,
+  COUNT(IF ( sore_throat = 'yes', 1, NULL)) AS sore_throat_yes,
+  COUNT(IF ( rhinitis = 'no', 1, NULL)) AS rhinitis_no,
+  COUNT(IF ( rhinitis = 'yes', 1, NULL)) AS rhinitis_yes,
+  COUNT(IF ( stomach_issues = 'no', 1, NULL)) AS stomach_issues_no,
+  COUNT(IF ( stomach_issues = 'yes', 1, NULL)) AS stomach_issues_yes,
+  COUNT(IF ( sensory_issues = 'no', 1, NULL)) AS sensory_issues_no,
+  COUNT(IF ( sensory_issues = 'yes', 1, NULL)) AS sensory_issues_yes,
+  COUNT(IF ( longterm_medication	 = 'no', 1, NULL)) AS longterm_medication_no,
```
Comment (nathandao) [inline src/backend/dataExports/postalCodeLevelGeneralResults.ts:89]: Same question about the large spaces here ...

File: src/backend/dataExports/cityLevelWeeklyGeneralResults.ts (hunk 2 above)
Comment (cutepig) [reply src/backend/dataExports/cityLevelWeeklyGeneralResults.ts:53]: I guess they are intentional. This was directly copy/pasted from the query Esa provided. I have in mind to improve the queries with a helper that would generate these fields (the low-level support exists in `stringLiteralUnionFields` function in `model.ts`).

Comment (nathandao) [reply src/backend/dataExports/cityLevelWeeklyGeneralResults.ts:53]: Ohhhh that'll be really neat!

I took a quick look at `stringLiteralUnionFields`, got dizzy before digging further :sweat_smile:  🥴

**Pull request merged**